Dockerfile

```diff
@@ -25,6 +25,9 @@ RUN [ -s /home/extractor/requirements.txt ] && \
     (echo "No python modules to install" && \
     rm /home/extractor/requirements.txt)
 
-USER extractor
 
 COPY *.py /home/extractor/
+RUN chmod a+x /home/extractor/transformer.py
+
+USER extractor
+ENTRYPOINT  ["/home/extractor/transformer.py"]
```

Comment:
This is a question rather than a comment. May I ask what creating the entrypoint does? Going off of what I found online it looks like it changes the entrypoint from /bin/sh -c to another, but I don't know what changing it means. When I search online it says that the ENTRYPOINT instruction configures a container that will run as an executable

Reply:
It what Docker will run by default - if no other entrypoint is defined on the command line

Reply:
Thank you